package.json

```diff
@@ -54,7 +54,7 @@
     "docs:build": "yarn build && vitepress build civet.dev",
     "docs:preview": "yarn build && vitepress preview civet.dev",
     "prepublishOnly": "yarn build && yarn test",
-    "test": "c8 mocha"
+    "test": "c8 mocha && tsc"
```

Comment:
@STRd6 This is going to interact with #638. But I guess we can always add to `exclude` until we're ready.

Reply:
#638 is now merged. Can you merge `main` and test what happens now? We might need to `exclude` it for now.